ios/Classes/MessagingInApp/CustomerIOInAppMessaging.swift

```diff
@@ -40,10 +40,23 @@ public class CustomerIOInAppMessaging: NSObject, FlutterPlugin {
     }
 
     func configureModule(params: [String: AnyHashable]) {
```

Comment:
lets add test for this method and I think that would suffice all concerns, of parsing

Reply:
We do have tests in flutter repo but they don't go till binding (swift) level, but I don't expect it to be hard either (I can be wrong). I think we have channels setup for test, we just need to see how can we listen to those test channels at native level.